test/governance/utils/Votes.behavior.js

```diff
@@ -245,95 +247,95 @@ function shouldBehaveLikeVotes () {
         expect(await this.votes.getPastTotalSupply(0)).to.be.bignumber.equal('0');
       });
 
-      it('returns the latest block if >= last checkpoint block', async function () {
-        const t1 = await this.votes.mint(this.account1, this.token0);
-        await time.advanceBlock();
+      it('generally returns the voting balance at the appropriate checkpoint #1', async function () {
```

Comment:
This test title can be improved. Why is there `#1` and `#2`? And note that this is testing for the total supply, not the balance.